### ISSUE TITLE
Change colors when customizing

### ISSUE THREAD

**Issue by urdadx** [WahabDev12/Gradients-Ninja#3]
Add functionality to change colors when customizing gradients in the editor

**Comment by PriyansuMaurya** [comment]
![image](https://github.com/urdadx/libre-gradients/assets/101447544/a0af67b3-5835-481b-92dc-275c93c5a970)

You mean user should be able to changes this input fields of colors.

**Comment by urdadx** [comment]
Yeah, exactly as you illustrated above. Also, the color could be an `<input type="color" />` to make changing colors easier. 

![image](https://github.com/urdadx/libre-gradients/assets/70736338/94e55187-5eb3-40f2-be0a-c189365a1709)

**Comment by PriyansuMaurya** [comment]
Assign this to me. I will do this in coming days.
